### ISSUE TITLE
why network seems to have a lot of false negatives ?

### ISSUE THREAD

**Issue by jediofgever** [dineshreddy91/Occlusion_Net#5]

**Comment by dineshreddy91** [comment]
Can you explain with some examples??

**Comment by moshanATucsd** [comment]
Hi, we tried your method on KITTI, and it seems not able to handle occluded cars, and the false positive (false car in middle) also exists 

![image](https://user-images.githubusercontent.com/23324577/69592428-b55f4c80-0faa-11ea-8605-164f7ed9227c.png)

**Comment by dineshreddy91** [comment]
This is a threshold problem in the detections from Maskrcnn(its currently 0.6 if you increase it the false positives in the middle will vanish). And the data we trained on is not for autonomous driving scenarios yet. Since we saw only occlusions with a maximum of two overlaps in the carfusion dataset we cannot yet fix multiple overlap of cars as you show in your example on the left. Improving the detector will help such situations.
